Gruntfile.js

```diff
@@ -4,7 +4,8 @@ module.exports = function(grunt) {
     "src/spec/payid-uri.md",
     "src/spec/payid-discovery.md",
     "src/spec/payid-protocol.md",
-    "src/spec/verifiable-payid-protocol.md"
+    "src/spec/verifiable-payid-protocol.md",
+    "src/spec/verifiable-payid-protocol-NC.md"
```

Comment:
Can we close this PR and move this change into https://github.com/payid-org/rfcs/pull/39? We also need to generate the `dist` artifacts here as part of the PR.